### ISSUE TITLE
Read about "Outliers"

### ISSUE THREAD

**Issue by VCF** [cherrypi/Science-Fair_2019#17]
* http://mathworld.wolfram.com/Outlier.html
* https://en.wikipedia.org/wiki/Outlier
* https://www.mathopenref.com/outlier.html

Consider this topic in the context of these plots (from 9f976f5):

* [rainfall vs surface area](https://github.com/cherrypi/Science-Fair_2019/blob/9f976f5f9b2edee73fa37cb4298fed8cd461eee2/Graphs/SPraZ.png)
* [rainfall vs depth](https://github.com/cherrypi/Science-Fair_2019/blob/9f976f5f9b2edee73fa37cb4298fed8cd461eee2/Graphs/SPrdZ.png)
* [rainfall vs volume](https://github.com/cherrypi/Science-Fair_2019/blob/9f976f5f9b2edee73fa37cb4298fed8cd461eee2/Graphs/SPrvZ.png)

Why do you think I brought this up?

Note that outliers are often searched for to identify "incorrect" data, but they can also just be "atypical" or "unsual". Or they can be real, and reality is not as homogenous as you had hoped for. However, it's often a practice to remove presumed outliers from data sets; If data are excluded from an analysis, it __must__ be noted, including your reasons for doing so. Whatever you do, it's well worth touching on outlier points in your discussion section, and probably in the results section as well.

**Comment by cherrypi** [comment]
You brought it up because of some of the more extreme rain measurements which stood apart from the typical 0.1 to 0,.5 measurements, I feel like I should still incorporate them, but if you don't agree with that that's okay to.

**Comment by VCF** [comment]
Yes, the issue is with the extreme values. But it's not that it's wrong. But whenever you identify what you think is an outlier, you should ask (or better, test) - "what happens to my results / figures / conclusions if the outlier(s) are removed?"
